runtime/src/ancient_append_vecs.rs

```diff
@@ -1155,62 +1156,76 @@ pub mod tests {
 
     #[test]
     fn test_write_ancient_accounts() {
-        for num_slots in 0..4 {
-            for combine_into in 0..=num_slots {
-                if combine_into == num_slots && num_slots > 0 {
-                    // invalid combination when num_slots > 0, but required to hit num_slots=0, combine_into=0
-                    continue;
-                }
-                let (db, storages, slots, _infos) = get_sample_storages(num_slots);
+        for data_size in [None, Some(10_000_000)] {
```

Comment:
this is a massive diff due to indentation. removing whitespace shows the interesting changes.